int32 MaxContacts
int32 BallNum
int32 BallType
float64 BallRad
float64 BallBoxRatio
float64 ViscosityParam1
float64 ViscosityMaxDist
float64 BoxThickness
float64 SrcSizeXY
float64 SrcSizeZ
float64 SrcSize2S
float64 SrcSize2H
float64[] RcvPos
float64[] RcvSize
float64 ContactBounce
float64 ContactBounceVel
float64 ContactSoftCFM
float64 GripperHeight
float64 TimeStep
float64 Gravity
